lib/paperclip/thumbnail.rb

```diff
@@ -76,7 +77,11 @@ def make
 
         parameters = parameters.flatten.compact.join(" ").strip.squeeze(" ")
 
-        success = convert(parameters, :source => "#{File.expand_path(src.path)}#{'[0]' unless animated?}", :dest => File.expand_path(dst.path))
+        success = convert(parameters, 
+          :source => "#{File.expand_path(src.path)}#{'[' + @frame_index.to_s + ']' unless animated?}", 
```

Comment:
@tute so this is interesting. I updated this line and it already had the 'unless animated?' check.  It used to specify a [0] index.  However in the tests I did, i do not pass in the animated param, and the class sets that value to true by default.  It does make sense that we should be only doing this if animated, but maybe this is a different bug?  any thoughts?